workbench-for-microsoft-azure-ml/Dockerfile.ubuntu2204

```
@@ -87,22 +87,14 @@ RUN apt-get update --fix-missing -qq \
     && rm -rf /var/lib/apt/lists/*
 
 ### Install Jupyter and extensions ###
-RUN /opt/python/"${PYTHON_VERSION_JUPYTER}"/bin/python -m venv /opt/python/jupyter \
+RUN RUN /opt/python/"${PYTHON_VERSION_JUPYTER}"/bin/python -m venv /opt/python/jupyter \
```

```suggestion
RUN /opt/python/"${PYTHON_VERSION_JUPYTER}"/bin/python -m venv /opt/python/jupyter \
```
I was excited to stumble upon this MR to add JL 4 and coincidentally noticed this - apologies for the unsolicited comment!